### PR TITLE
Projects popup

### PR DESCRIPTION
**Pull request by AhmedHossam01**
Create projects popup

### CODE REVIEW

File: projects.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    company: "Tonic",
+    position: "Back End Dev",
+    date: "2015",
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required.",
+    body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,",
+    languages: ["html", "css", "javascript"],
+    imageUrl: "images/project2.png",
+    id: "2",
+  },
+];
```
Comment (rustam-tolipov) [inline projects.js]: 😀GOOD JOB GUYS 🙌😀
- [x] Please add 2 more projects to make it perfect, ok 👍? _The more projects you have, the easier it is to know you and your experience well._ Will all these steps your project should be completed 🍀

File: index.html [hunk 2]
```diff
@@ -78,79 +78,7 @@ <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
   </section>
 
   <section class="works" id="portfolio">
```
Comment (rustam-tolipov) [inline index.html:80]: - [x] Please also make sure your design follows the Figma design. I would recommend making them in a `Z` pattern, however, you have used the `E` pattern. You may see the difference from the screenshots down below. 🙂📸

Your design

Screen Shot 2022-01-26 at 10 17 37 PM


Figma design

image

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (mateo951) [reply index.html:80]: Addressed 😎

File: projects.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    company: "Tonic",
+    position: "Back End Dev",
+    date: "2015",
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required.",
+    body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,",
+    languages: ["html", "css", "javascript"],
+    imageUrl: "images/project4.png",
+    id: "4",
+  },
+];
```
Comment (mateo951) [inline projects.js:46]: Addressed 👍

**Review by mateo951 — APPROVED** [review]
### Hi Team,

Good job making the changes requested by the previous reviewer 😎
Your project is complete!  There is nothing else to say other than... it's time to merge it 😉 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**